scripts/generate-genesis.py

```diff
@@ -0,0 +1,284 @@
+    self.l2_allocs_path = os.path.join(supersim_output_dir, 'l2-allocs', f"{self.l2_chain_id}-l2-allocs.json")
+    self.l2_genesis_path = os.path.join(supersim_output_dir, 'l2-genesis', f"{self.l2_chain_id}-l2-genesis.json")
+    self.l2_rollup_config_path = os.path.join(supersim_output_dir, 'rollup-configs', f"{self.l2_chain_id}-rollup-config.json")
+
+class Bunch:
+    def __init__(self, **kwds):
+        self.__dict__.update(kwds)
+
+def main():
+  supersim_dir = os.path.abspath(os.path.join(os.path.realpath(__file__), '../..'))
+  supersim_contracts_dir = os.path.join(supersim_dir, 'contracts')
+  supersim_output_dir = os.path.join(supersim_dir, 'genesisdeployment', 'generated')
+  supersim_output_deploy_configs_dir = os.path.join(supersim_output_dir, "deploy-configs")
+  supersim_output_l1_allocs_dir = os.path.join(supersim_output_dir, "l1-allocs")
+  supersim_output_addresses_dir = os.path.join(supersim_output_dir, "addresses")
+  supersim_output_l2_allocs_dir = os.path.join(supersim_output_dir, "l2-allocs")
+  supersim_output_l2_genesis_dir = os.path.join(supersim_output_dir, "l2-genesis")
+  supersim_output_rollup_dir = os.path.join(supersim_output_dir, "rollup-configs")
+
+  os.makedirs(supersim_output_dir, exist_ok=True)
+  os.makedirs(supersim_output_deploy_configs_dir, exist_ok=True)
+  os.makedirs(supersim_output_l1_allocs_dir, exist_ok=True)
+  os.makedirs(supersim_output_addresses_dir, exist_ok=True)
+  os.makedirs(supersim_output_l2_allocs_dir, exist_ok=True)
+  os.makedirs(supersim_output_l2_genesis_dir, exist_ok=True)
+  os.makedirs(supersim_output_rollup_dir, exist_ok=True)
+
+  monorepo_dir = os.path.abspath(monorepo_base_path)
+  monorepo_op_node_dir = os.path.join(monorepo_dir, 'op-node')
+  monorepo_contracts_bedrock_dir = os.path.join(monorepo_dir, 'packages', 'contracts-bedrock')
+  monorepo_deployment_dir = os.path.join(monorepo_contracts_bedrock_dir, 'deployments')
+  monorepo_deploy_config_dir = os.path.join(monorepo_contracts_bedrock_dir, 'deploy-config')
+  monorepo_forge_l1_dump_path = os.path.join(monorepo_contracts_bedrock_dir, 'state-dump-900.json')
+  monorepo_default_deploy_config_template_path = os.path.join(monorepo_deploy_config_dir, 'devnetL1-template.json')
+
+  paths = Bunch(
+    supersim_dir=supersim_dir,
+    supersim_contracts_dir=supersim_contracts_dir,
+
+    monorepo_dir=monorepo_dir,
+    monorepo_op_node_dir=monorepo_op_node_dir,
+    monorepo_contracts_bedrock_dir=monorepo_contracts_bedrock_dir,
+    monorepo_deployment_dir=monorepo_deployment_dir,
+    monorepo_deploy_config_dir=monorepo_deploy_config_dir,
+    monorepo_forge_l1_dump_path=monorepo_forge_l1_dump_path,
+    monorepo_default_deploy_config_template_path=monorepo_default_deploy_config_template_path,
+  )
+
+  l1_chain_id = 900
+
+  def get_l2_chain(i): 
+    return L2(l1_chain_id, 901 + i, deployer_addresses[i], supersim_dir)
+
+  l1_chain = L1(l1_chain_id, supersim_dir)
+  l2_chains = list(map(get_l2_chain, range(10)))
```

Comment:
do you think we can reduce this down to 3-5 just so it's less json files we need to check in?

Reply:
yep! will bring down to 5. it will also bloat our binary size